### PR TITLE
Update regular page

### PR DESCRIPTION
**Pull request by lingeshwaran05**
## 🛠️ Fixes Issue

#236 

## 👨‍💻 Changes proposed

updated regular page upto dec 22

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

### CODE REVIEW

File: pages/Regular.md [hunk 1]
```diff
@@ -3,6 +3,34 @@
...
+|15 Oct 2022|![image](https://user-images.githubusercontent.com/76167753/209154758-0125c5e7-c272-4dc0-9086-a73d16d3e6c1.png)|[`Post Link`](https://twitter.com/pradumna_saraf/status/1581125072143556608?s=20&t=_eF7n2kPp9v8V5Eja0TGCg)|**Pradumna Saraf** |
+|12 Oct 2022|![image](https://user-images.githubusercontent.com/76167753/209154598-59899512-e54b-4bd1-adca-ec816abc6363.png)|[`Post Link`](https://twitter.com/pradumna_saraf/status/1580022691746152450?s=20&t=_eF7n2kPp9v8V5Eja0TGCg)|**Pradumna Saraf** |
+|08 Oct 2022|![image](https://user-images.githubusercontent.com/76167753/209154207-106e62d4-497b-4ffc-9c67-bba423a92cf0.png)|[`Post Link`](https://twitter.com/pradumna_saraf/status/1578573186190479361?s=20&t=_eF7n2kPp9v8V5Eja0TGCg)|**Pradumna Saraf** |
+|06 Oct 2022|!![image](https://user-images.githubusercontent.com/76167753/209153790-2b5505a8-e119-4338-b697-4e3d4f882b2b.png)|[`Post Link`](https://twitter.com/pradumna_saraf/status/1577848389160542209?s=20&t=_eF7n2kPp9v8V5Eja0TGCg)|**Pradumna Saraf** |
```
Comment (Pradumnasaraf) [inline pages/Regular.md:33]:
```suggestion
|06 Oct 2022|![image](https://user-images.githubusercontent.com/76167753/209153790-2b5505a8-e119-4338-b697-4e3d4f882b2b.png)|[`Post Link`](https://twitter.com/pradumna_saraf/status/1577848389160542209?s=20&t=_eF7n2kPp9v8V5Eja0TGCg)|**Pradumna Saraf** |
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

 Just need to remove the extra exclamation mark. Also, can you add the ALT text? in place of `[image]`

File: pages/Regular.md (hunk 1 above)
Comment (lingeshwaran05) [reply pages/Regular.md:33]: yeah sure , thanks for the review

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**